index.bs

```diff
@@ -656,32 +756,59 @@ precomposed character).</p>
 
 
 <h2 id=data-model>Data model</h2>
-<!-- Add some content here about cues and serialisation format in general -->
 <!-- Describe metadata, caption/subtitle, chapter & description cues -->
 
+<h3 id=model-overview>Overview</h3>
 
-<h3 id=cues>WebVTT cues</h3>
+<p><i>This section is non-normative.</i></p>
 
-<p>A <dfn>WebVTT cue</dfn> is a <a>text track cue</a> that additionally consist of the following:
-[[!HTML]]</p>
+<p>The WebVTT file is a container file for chunks of data that are time-aligned with a video or
+audio resource. It can therefore be regarded as a serialisation format for time-aligned data.</p>
+
+<p>A WebVTT file starts with a header and then contains a series data blocks. If a data block has a
+start and end time, it is called a WebVTT cue. A comment is another kind of data block.</p>
+
+<p>A WebVTT file can contain cues with different kinds of data. The HTML specification identifies
+captions, subtitles, chapters, audio descriptions and metadata as data kinds.</p>
+
+<p>A WebVTT file only contains data of one kind, never a mix of different kinds of data.</p>
```

Comment:
Make it a statement of fact "since the kind of a file is externally identified and applies to the whole file, there is no mechanism currently for making a file of mixed kinds"?

Reply:
We need this as a normative requirement, because anything else causes mayhem - we'd need to associate a "kind" with every cue. From an authoring and process point of view, mixed files would cause all kinds of headaches. I'll make this statement stronger.